run_infra_tests.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+while [[ $# -gt 0 ]]
```

Comment:
It doesn't seem like you are passing any args from https://github.com/fabric8-hdd/supervisor/pull/46/files#diff-3b49669746d5c8707421c471bc8a977cR43